Task2/stack.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include <stdlib.h>
+#include <stdio.h>
+#include <stdbool.h>
+
+typedef struct StackElement
+{
+    int value;
+    struct StackElement* nextElementPtr;
+} StackElement;
+
+typedef struct Stack
+{
+    StackElement* lastElementPtr;
+} Stack;
```

Comment:
```suggestion
typedef struct Stack Stack;
```
Плохая практика оставлять пользователю типа данных прямой доступ к внутреннему представлению этого типа.
В заголовочном файле должно быть только определение типа. А вот в реализации тип должен быть описан полностью.
(-0.5 балла)